bundles/org.openhab.binding.dominoswiss/src/main/java/org/openhab/binding/dominoswiss/internal/dominoswissHandlerFactory.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (c) 2010-2021 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.dominoswiss.internal;
+
+import static org.openhab.binding.dominoswiss.internal.dominoswissBindingConstants.*;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.openhab.core.thing.Thing;
+import org.openhab.core.thing.ThingTypeUID;
+import org.openhab.core.thing.binding.BaseThingHandlerFactory;
+import org.openhab.core.thing.binding.ThingHandler;
+import org.openhab.core.thing.binding.ThingHandlerFactory;
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * The {@link DominoswissHandlerFactory} is responsible for creating things and thing
+ * handlers.
+ *
+ * @author Frieso Aeschbacher - Initial contribution
+ */
+@NonNullByDefault
+@Component(configurationPid = "binding.dominoswiss", service = ThingHandlerFactory.class)
+public class dominoswissHandlerFactory extends BaseThingHandlerFactory {
+
+    private static final List<ThingTypeUID> SUPPORTED_THING_TYPES_UIDS = Arrays.asList(DOMINOSWISSEGATE_THING_TYPE,
+            DOMINOSWISSBLINDS_THING_TYPE);
```

Comment:
Better keep it consistent with other bindings.
```suggestion
    private static final Set<ThingTypeUID> SUPPORTED_THING_TYPES_UIDS = Set.of(DOMINOSWISSEGATE_THING_TYPE,
            DOMINOSWISSBLINDS_THING_TYPE);
```